data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -171,14 +171,25 @@
   </ntp-client>
   <partitioning config:type="list">
     <drive>
-  % my $wwn = {lipa => 'wwn-0x5002538c4002f8bd', briza => 'wwn-0x5000c5004f0e566d', pegasus => 'wwn-0x5002538c4002f8be', galactica => 'wwn-0x5002538c4002f8cd'};
+  % my $wwn = {lipa => 'wwn-0x5002538c4002f8bd', briza => 'wwn-0x5000c5004f0e566d', pegasus => 'wwn-0x5002538c4002f8be', galactica => 'wwn-0x5002538c4002f8cd', 'bare-metal1' => 'nvme-eui.35354b3054a009160025384300000002', 'bare-metal2' => 'nvme-eui.35354b3054a008260025384300000002', 'bare-metal3' => 'nvme-eui.35354b3054a008880025384300000002'};
   % my $hostname = (split(/\./, $get_var->("SUT_IP")))[0];
   % my $device_id = defined($wwn->{$hostname}) ? '/dev/disk/by-id/' . $wwn->{$hostname} : '';
       <device><%= $device_id %></device>
       <disklabel>gpt</disklabel>
       <enable_snapshots config:type="boolean">false</enable_snapshots>
       <initialize config:type="boolean">true</initialize>
       <partitions config:type="list">
+        % if ($check_var->('IPXE_UEFI', '1') or $check_var->('UEFI', '1')) {
```

Comment:
@RoyCai7 FYI, this PR will support the host installation works for uefi boot machines.